Packs/Cymulate/Integrations/Cymulate_v2/Cymulate_v2.py

```diff
@@ -0,0 +1,1791 @@
+                                      url_suffix=f'/{endpoint}/ids',
+                                      headers=self.headers)
+        response = response.get('data') if response else response
+        return response
+
+    def get_attack_by_id(self, endpoint: Optional[str], attack_id: Optional[str]):
+        """Retrieves data regarding an attack by the attack ID.
+
+        Args:
+            endpoint (str): The Cymulate endpoint to retrieve attacks data from.
+            attack_id (str): ID of the attack to retrieve data to.
+        """
+        response = self._http_request(method='GET',
+                                      url_suffix=f'/{endpoint}/attack/technical/{attack_id}',
+                                      headers=self.headers)
+        response = response.get('data') if response else response
+        return response
+
+    def get_immediate_threat_assessment(self, attack_id: Optional[str]):
+        """Retrieves data regarding immediate threats attack by the attack ID.
+
+        Args:
+            attack_id (str): ID of the attack to retrieve data to.
+        """
+        response = self._http_request(method='GET',
+                                      url_suffix=f'/immediate-threats/attack/technical/{attack_id}',
+                                      headers=self.headers).get('data')
+        response = response.get('payloads') if response else response
+        return response
+
+    def get_simulations_by_id(self, endpoint: Optional[str], attack_id: Optional[str]):
+        """ Retrieves all event data."""
+        return self._http_request(method='GET',
+                                  url_suffix=f'{endpoint}/history/technical/{attack_id}',
+                                  headers=self.headers,
+                                  params={'id': attack_id})
+
+
+''' HELPER FUNCTIONS '''
+
+
+def extract_status_commands_output(result: dict) -> dict:
+    """Parse the dictionary returned from the API call to a XSOAR output.
+
+    Args:
+        result (dict): API call result.
+    """
+    output = {
+        'id': result.get('id') if result.get('id') else result.get('_id'),
+        'inProgress': result.get('inProgress'),
+        'progress': result.get('progress'),
+        'addresses': result.get('addresses'),
+        'categories': result.get('categories'),
+    }
+    return output
+
+
+def build_data(params: dict) -> str:
+    """ Helper function to build HTTP request parameters. Cymulate's API does not accept Python's
+     'True' and 'False' objects, it requires a true and false objects (lowercase t and f).
+     In addition this function converts the dictionary to a string.
+
+    Args:
+        params (dict): parameter dictionary send to the API.
+
+    Returns:
+        str: params as a string, with lower case boolean values.
+    """
+    data = json.dumps(params).replace('False', 'false')
+    data.replace('False', 'false')
+    data.replace('True', 'true')
```

Comment:
`replace` doesn't modify the string, it returns a new object.
if the boolean values are string-based, this will work.
```suggestion
    data = json.dumps(params).replace('False', 'false').replace('True', 'true')
```
otherwise, using the `json.dumps` function should be enough:
![image](https://user-images.githubusercontent.com/30797606/110260474-1b5fbf80-7fb5-11eb-9eb2-2f777a2d8f7b.png)